### PR TITLE
Add Ore Beta Link

### PR DESCRIPTION
**Pull request by phase**
We're planning on merging this next weekend (July 14th, 2018).

PRs that need to be merged:
* [Ore](https://github.com/SpongePowered/Ore/pull/560)
* [SpongeHome](https://github.com/SpongePowered/SpongeHome/pull/146)
* [sponge-docs-theme](https://github.com/SpongePowered/sponge-docs-theme/pull/22)
* [SpongeDownloads](https://github.com/SpongePowered/SpongeDownloads/pull/5)
* [discourse-spongepowered-navbar](https://github.com/lukegb/discourse-spongepowered-navbar/pull/6)
* [SpongeAuth](https://github.com/SpongePowered/SpongeAuth/pull/193)

---

I did not add the Ore link to the section below the header because Bootstrap grids have a width of 12 units and upping the count from 6 objects of length 2 to 7 objects of length ?? is difficult. @Owexz advised not to worry about it right now.

Temp image but this is the same HTML
temp

### CODE REVIEW

File: src/html/include/base.html [hunk 1]
```diff
@@ -67,6 +67,7 @@
                     <li><a href="https://docs.spongepowered.org"><i class="fa-fw fa fa-book"></i>Docs</a></li>
                     <li><a href="https://jd.spongepowered.org"><i class="fa-fw fa fa-graduation-cap"></i>Javadocs</a></li>
                     <li><a href="https://forums.spongepowered.org/c/plugins/plugin-releases"><i class="fa-fw fa fa-plug"></i>Plugins</a></li>
+                    <li><a href="https://ore.spongepowered.org/"><i class="fa fa-cubes">Ore <b>Beta</b></i></a></li>
```
Comment (phit) [inline src/html/include/base.html]: `</i>` needs to be before the title

Comment (phase) [reply src/html/include/base.html]: fixed

**Comment by phase** [comment]
The ore logo will be at `spongepowered.org/assets/img/icons/ore.svg`

File: src/html/include/base.html [hunk 2]
```diff
@@ -67,7 +67,7 @@
                     <li><a href="https://docs.spongepowered.org"><i class="fa-fw fa fa-book"></i>Docs</a></li>
                     <li><a href="https://jd.spongepowered.org"><i class="fa-fw fa fa-graduation-cap"></i>Javadocs</a></li>
                     <li><a href="https://forums.spongepowered.org/c/plugins/plugin-releases"><i class="fa-fw fa fa-plug"></i>Plugins</a></li>
-                    <li><a href="https://ore.spongepowered.org/"><i class="fa fa-cubes"></i>Ore <b>Beta</b></a></li>
+                    <li><a href="https://ore.spongepowered.org/"><img src="assets/img/icons/ore.svg" alt="" style="height: 15px" class="fa"> Ore <sup>Beta</sup></a></li>
```
Comment (kashike) [inline src/html/include/base.html]: nooooo. no inline style

**Review by kashike — CHANGES_REQUESTED** [review]

**Review by lol768 — COMMENTED** [review]
CSP

File: src/html/include/base.html [hunk 3]
```diff
@@ -67,6 +67,7 @@
                     <li><a href="https://docs.spongepowered.org"><i class="fa-fw fa fa-book"></i>Docs</a></li>
                     <li><a href="https://jd.spongepowered.org"><i class="fa-fw fa fa-graduation-cap"></i>Javadocs</a></li>
                     <li><a href="https://forums.spongepowered.org/c/plugins/plugin-releases"><i class="fa-fw fa fa-plug"></i>Plugins</a></li>
+                    <li><a href="https://ore.spongepowered.org/"><img src="assets/img/icons/ore.svg" alt="" style="height: 15px" class="fa"> Ore <sup>Beta</sup></a></li>
```
Comment (lol768) [inline src/html/include/base.html]: Pls no `unsafe-inline` `style-src`

File: src/html/include/base.html [hunk 4]
```diff
@@ -67,6 +67,7 @@
                     <li><a href="https://docs.spongepowered.org"><i class="fa-fw fa fa-book"></i>Docs</a></li>
                     <li><a href="https://jd.spongepowered.org"><i class="fa-fw fa fa-graduation-cap"></i>Javadocs</a></li>
                     <li><a href="https://forums.spongepowered.org/c/plugins/plugin-releases"><i class="fa-fw fa fa-plug"></i>Plugins</a></li>
+                    <li><a href="https://ore.spongepowered.org/"><img src="assets/img/icons/ore.svg" alt="" class="fa-fw fa ore-nav">Ore <sup>Beta</sup></a></li>
```
Comment (kashike) [inline src/html/include/base.html:70]: Did you mean `ore-logo`?

Comment (phase) [reply src/html/include/base.html:70]: My bad, some repos already use `ore-logo` for a separate Ore logo.

**Pull request merged**